cudf/tests/test_dataframe.py

```diff
@@ -356,6 +356,34 @@ def test_dataframe_to_string_wide():
     assert got.split() == expect.split()
 
 
+def test_dataframe_empty_to_string():
+    # Test for printing empty dataframe
+    df = DataFrame()
+    got = df.to_string()
+    print(got)
+    expect = '''
```

Comment:
Please correct the indentation.
```suggestion
    expect = "Empty DataFrame\nColumns: []\nIndex: []\n"
```

Reply:
corrected indentation.